File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
         bool val=false;
        int cnt=0;
        for(int i=s.length()-1;i>=0;i--)
        {
            if(s[i]==' '){
                if(val)
                break;
            }
            else{
                 val=true;
                cnt++;
            }
        }
        return cnt;
    }
};

File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        String arr[] = s.split(" ");
        int n=arr.length;
         return arr[n-1].length();
        // for(int i=arr.length-1;i>=0;i--){
        //     if(!arr[i].isEmpty()){
        //         return arr[i].length();
        //     }
        // }

       
    }
}
